lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -952,6 +952,16 @@ def split_colon_ipv6(input, module):
     return result
 
 
+def detect_ipvX_address_usage():
```

Comment:
It's better to define this one inline shortly before `option_minimal_versions`, so you have access to `client.module.params`.